sessionctx/variable/removed.go

```diff
@@ -21,11 +21,30 @@ package variable
 // This helps ensure some compatibility for applications while being
 // careful not to return dummy data.
 
+const (
+	tiDBEnableAlterPlacement       = "tidb_enable_alter_placement"
+	tiDBMemQuotaHashJoin           = "tidb_mem_quota_hashjoin"
+	tiDBMemQuotaMergeJoin          = "tidb_mem_quota_mergejoin"
+	tiDBMemQuotaSort               = "tidb_mem_quota_sort"
+	tiDBMemQuotaTopn               = "tidb_mem_quota_topn"
+	tiDBMemQuotaIndexLookupReader  = "tidb_mem_quota_indexlookupreader"
+	tiDBMemQuotaIndexLookupJoin    = "tidb_mem_quota_indexlookupjoin"
+	tiDBEnableGlobalTemporaryTable = "tidb_enable_global_temporary_table"
+	tiDBSlowLogMasking             = "tidb_slow_log_masking"
+	placementChecks                = "placement_checks"
+)
```

Comment:
These were moved in here from `sessionctx/variable/tidb_vars.go`. The comments around them said they were deprecated, but they've been removed. So isolating them to here and no longer exporting seems to be a better fit to truly ensure this is the case.